### DISCUSSION TITLE
Roadmap schedule question

### DISCUSSION THREAD

**Discussion by ha1fstack** [geldata/gel#5093] [General]
Thanks edgedb team for this awesome project!

My question is, is there any target schedule for planned features below:
- Materialized view
- Query cache
- Migration from existing postgres based project
- Read replica support

Thank you in advance for the reply.

**Comment by raddevon** [comment]
- Materialized view & query cache- We are working on these. We don't have an ETA to share yet.
- Migration from existing postgres based project- It's not possible to do this in a generic way that adapts any possible Postgres schema to an EdgeDB schema. We hope to do something like this for Prisma schemas since we can cleanly map them to EdgeDB schemas. You could do it yourself with a script custom built for your schema if manually adapting the schema is impractical.
- Read replica support- Support is planned for 3.0, but implementation in the client bindings will take some time. May not be usable day-one of the 3.0 launch as a result.
